src/applications/personalization/profile/components/ProfileInfoTable.jsx

```diff
@@ -106,7 +107,12 @@ const ProfileInfoTable = ({
           )
           .map((row, index) => (
             // eslint-disable-next-line jsx-a11y/no-redundant-roles
-            <li key={index} className={classes.tableRow} role="listitem">
+            <li
+              key={index}
+              className={classes.tableRow}
+              role="listitem"
+              id={row.id}
```

Comment:
if the row data includes an id, use it

Reply:
Is there a chance the row data would not include an id? If so, does this list item still load?

Reply:
it just leaves the `id` attribute off of the `li`